gravitee-apim-portal-webui/src/app/components/gv-alert/gv-alert.component.ts

```diff
@@ -165,6 +175,12 @@ export class GvAlertComponent implements OnInit {
     }
   }
 
+  toggleWebhook({ detail }) {
```

Comment:
Type? It's a boolean I guess 🤷🏻 ?

```suggestion
  toggleWebhook({ detail }: { detail: boolean }) {
```